src/gutenberg/components/typography/index.jsx

```diff
@@ -0,0 +1,292 @@
+        label = __( 'Black Italic' );
+        break;
+    }
+
+    return label;
+}
+
+/**
+ * Get all font widths.
+ *
+ * @param {string} font - search font.
+ * @param {string} fontFamilyCategory - font fontFamilyCategory.
+ * @return {Array} - all font widths.
+ */
+function getFontWeights( font, fontFamilyCategory ) {
+    const {
+        fonts,
+    } = GHOSTKIT;
+
+    const fontWeights = [];
+
+    if ( font !== '' && fontFamilyCategory !== '' && typeof font !== 'undefined' && typeof fontFamilyCategory !== 'undefined' ) {
+        Object.keys( fonts[ fontFamilyCategory ].fonts ).forEach( ( fontKey ) => {
+            if ( fonts[ fontFamilyCategory ].fonts[ fontKey ].name === font ) {
+                Object.keys( fonts[ fontFamilyCategory ].fonts[ fontKey ].widths ).forEach( ( widthKey ) => {
+                    const width = fonts[ fontFamilyCategory ].fonts[ fontKey ].widths[ widthKey ];
+                    fontWeights.push(
+                        { value: width, label: getFontWeightLabel( width ) }
+                    );
+                } );
+            }
+        } );
+    }
+
+    return fontWeights;
+}
+
+/**
+ * Component Class
+ */
+export default class Typorgaphy extends Component {
+    render() {
+        const {
+            onChange,
+            placeholders,
+            label,
+            fontFamily,
+            fontFamilyCategory,
+            fontWeight,
+            fontSize,
+            lineHeight,
+            letterSpacing,
+            childOf,
+            fontWeights = getFontWeights( fontFamily, fontFamilyCategory ),
+        } = this.props;
+
+        const fontFamilyValue = { value: fontFamily, label: fontFamily, fontFamilyCategory: fontFamilyCategory };
+        const fontWeightValue = { value: fontWeight, label: getFontWeightLabel( fontWeight ) };
+        const fontSizeValue = typeof fontSize === 'undefined' ? '' : fontSize;
+        const childOfClass = childOf !== '' ? ' ghostkit-typography-child' + ' ghostkit-typography-child-of-' + childOf : '';
+
+        return (
+            <div className={ 'ghostkit-typography' + childOfClass }>
+                <h4>{ label }</h4>
+                <div className="ghostkit-control-typography">
+                    {
+                        typeof fontFamily !== 'undefined' &&
+                        <div className="ghostkit-typography-font-control">
+                            <Tooltip text={ __( 'Font Family' ) }>
+                                <div>
+                                    <Select
+                                        value={ fontFamilyValue }
+                                        onChange={ ( opt ) => {
+                                            onChange( {
+                                                fontFamily: opt.value,
+                                                fontFamilyCategory: opt.fontFamilyCategory,
+                                                fontWeight: opt.value === 'Default Site Font' ? '' : '400',
```

Comment:
We can't use `Default Site Font` in conditions and save this value in DB because it is translatable Label for Select component. The value should be an empty string.